user/running-build-in-debug-mode.md

```diff
@@ -205,3 +205,17 @@ Press `q` to exit the log scroll mode.
 Once you exit from all the live `tmate` windows, the debug VM will terminate
 after resetting the job's status to the original status before you restarted it.
 No more phases (`before_install`, `install`, etc.) will be executed.
+
+## Known issues
+
+### Node.js debug builds need to set up the execution environment
```

Comment:
And it might be worth avoiding execution environment here.

While I'm being picky, builds in the title, and jobs in the body might trip people up.

And finally. Is it a debug build, or is it debug access to a Node.js build?

> Need to set up node and npm when debugging Node.js builds

or 

> Setting up Node.js on debug builds